.gitignore

```diff
@@ -1,6 +1,9 @@
 *~
 *.pyc
-work/
```

Comment:
Probably want to keep `work/` here if we're not changing the defaults yet?

Reply:
src/work is the correct directory, work/ is currently empty.